engine/release-notes/index.md

```diff
@@ -13,896 +13,164 @@ redirect_from:
+- Removed the deprecated `overrideKernelCheck` option from the `overlay2` storage driver. [moby/moby#44279](https://github.com/moby/moby/pull/44279)
+- Removed support for the deprecated `io.containerd.runtime.v1.linux` OCI runtime. [moby/moby#43695](https://github.com/moby/moby/pull/43695)
+- Removed LCOW (Linux Containers on Windows). [moby/moby#42451](https://github.com/moby/moby/pull/42451), [moby/moby#42499](https://github.com/moby/moby/pull/42499), [moby/moby#42506](https://github.com/moby/moby/pull/42506), [moby/moby#42511](https://github.com/moby/moby/pull/42511), [moby/moby#42520](https://github.com/moby/moby/pull/42520), [moby/moby#42683](https://github.com/moby/moby/pull/42683), [moby/moby#42684](https://github.com/moby/moby/pull/42684), [moby/moby#42685](https://github.com/moby/moby/pull/42685), [moby/moby#43187](https://github.com/moby/moby/pull/43187)
+
+  LCOW was introduced as a technical preview in 17.09 and deprecated in 20.10.
+
+- Removed daemon options related to legacy overlay networks, which were used with "standalone" Swarm.
+  - Removed `--cluster-xx` options from `dockerd`. [moby/moby#40383](https://github.com/moby/moby/issues/40383)
+  - Removed `host-discovery` and overlay networks with external k/v stores. [moby/moby#42247](https://github.com/moby/moby/pull/42247)
+- Removed a deprecated `arm` platform fallback. `--platform linux/arm/vY` will now return a error when `arm/vY` is not available instead of pulling the wrong image. [moby/moby#44414](https://github.com/moby/moby/pull/44414)
+- Removed the deprecated `SetCustomHTTPHeaders()`, `CustomHTTPHeaders()` options-setters from the Go client API. [moby/moby#42694](https://github.com/moby/moby/pull/42694)
+- Removed the deprecated `WithDialer()` option-setter from the Go client API. [moby/moby#44022](https://github.com/moby/moby/pull/44022)
+  
+  Users should use `WithDialContext()` instead.
+
+- Removed the daemon implementation of `opts.QuotedString`, which was moved to CLI. [moby/moby#43250](https://github.com/moby/moby/pull/43250)
+- Removed separate daemon ID from trust-key in the daemon, and disable generating the trust-key. [moby/moby#43555](https://github.com/moby/moby/pull/43555)
+- API: Removed the deprecated `KernelMemory` option from `POST /containers/create` on API >= 1.42. [moby/moby#43214](https://github.com/moby/moby/pull/43214)
+
+### Deprecated
+
+- Require Windows Server RS5 / LTSC 2019 (build 17763) as the minimum to run the daemon. [moby/moby#43254](https://github.com/moby/moby/pull/43254)
+- Deprecated `BuilderSize` in API versions >= 1.42 [moby/moby#42608](https://github.com/moby/moby/pull/42608)
+- Deprecated `BuildCache.Parent` in favor of the newly introduced `BuildCache.Parents` in API >= v1.42. [moby/moby#43908](https://github.com/moby/moby/pull/43908)
+- Deprecated `pkg/urlutil`, moving the implementation to `builder/remotecontext/urlutil`. [moby/moby#43477](https://github.com/moby/moby/pull/43477)
 
 ### Updates
```

Comment:
```suggestion
### Packaging
```

Reply:
Seems like the intention might be not renaming this section to stay consistent with other projects, @dvdksn? I think this is still more natural (as we can move the systemd change here), but feel free to resolve this and `s/Packaging/Updates` in my other comments if you think this should stay.

Reply:
We're trying to keep the release notes aligned across different products, hence my hesitation to change this to Packaging. Though I am not sure I like "Updates" either, so I am changing it to "Upgrades" now.